webapi/src/main/scala/org/knora/webapi/messages/app/appmessages/ApplicationStateMessages.scala

```diff
@@ -35,6 +35,15 @@ case class GetPrintConfigExtendedState() extends ApplicationStateRequest
 case class SetAppState(value: AppState) extends ApplicationStateRequest
 case class GetAppState() extends ApplicationStateRequest
 
+case class InitStartUp(withOntologies: Boolean = false) extends ApplicationStateRequest
```

Comment:
These class names are pretty self-explanatory, but for newcomers to the code, I think it would help to have a Scaladoc comment for each class.